pygpu/collectives.pyx

```diff
@@ -248,7 +256,8 @@ cdef class GpuComm:
         """
         if root == -1:
             comm_get_rank(self, &root)
-        return comm_broadcast(self, array, root)
+        comm_broadcast(self, array, root)
+        return
```

Comment:
Ok!
On Jul 8, 2016 19:29, "abergeron" <notifications@github.com> wrote:

> In pygpu/collectives.pyx
> <https://github.com/Theano/libgpuarray/pull/206#discussion_r70102148>:
>
> > @@ -248,7 +256,8 @@ cdef class GpuComm:
> >          """
> >          if root == -1:
> >              comm_get_rank(self, &root)
> > -        return comm_broadcast(self, array, root)
> > +        comm_broadcast(self, array, root)
> > +        return
>
> This is implicit in python, so no need to add it explicitly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/libgpuarray/pull/206/files/dc42c5388270fadf10c9bc60588cb44e580c5825..93c5568b34db1cf5f079b91c616942e5e747c1ed#r70102148>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD5VAXzZuPku3m8zqlqSG7lV2P9CQ6O8ks5qTnrmgaJpZM4JAkNl>
> .
>